infra/common/network/virtual-network.parameters.json

```diff
@@ -16,7 +16,7 @@
           "name": "#{{ networkResourceNamePrefix }}#{{ nc_resource_subnet }}#{{ nc_instance_regionid }}01",
           "addressPrefix": "#{{ subnet1AddressPrefix }}",
           "serviceEndpoints": [],
-          "routeTableId": "/subscriptions/#{{ subscriptionId }}/resourceGroups/#{{ virtualNetworkResourceGroup }}/providers/Microsoft.Network/routeTables/UDR-Spoke-Route-From-#{{ networkResourceNamePrefix }}#{{ nc_resource_virtualnetwork }}#{{ nc_instance_regionid }}01",
+          "routeTableId": "/subscriptions/#{{ subscriptionId }}/resourceGroups/#{{ virtualNetworkResourceGroup }}/providers/Microsoft.Network/routeTables/UDR-Spoke-Route-From-#{{ networkResourceNamePrefix }}#{{ nc_resource_virtualnetwork }}#{{ nc_instance_regionid }}01-AKS",
```

Comment:
what actually happens with the tables ending on "01" ?

Reply:
Hey Dan. The route table ending 01 is used for the remaining subnets (1495-1499). This is the route table that the additional rules are added too (Active_Directory_to_Internet & Postgres_To_Vnet). Happy to discuss further if needed

Reply:
This looks fine... This is just the link rather than RT creations. Some subnets retain the link to the 01. The AKS ones move to AKS subnets. LGTM